### ISSUE TITLE
Progress indicator when verification is running

### ISSUE THREAD

**Issue by franck44** [DafnyVSCode/ide-vscode#23]
Thanks for the update to the plugin.
There used to be a progress indicator displaying the number of queued queries. It was useful to see something wa happening.

it seems to have been disabled in the new version of the plugin.
It is now hard to tell whether something (verification) is happening.
It may occur that hundreds of queries are running (because the files are large, the VC is complex) and the progress indicator was useful to stop the verification and reload the window (otherwise verification slower and slower ...)

Is there a way the progress indicator could be restored? Or is it due to the update in Dafny 3?

**Comment by camrein** [comment]
This is not a limitation of Dafny; it just hasn't been integrated into the language server and the VSCode extension at this time.
I initially thought about removing the automatic verification from the language server and letting the plugin initiate the verification instead. However, this increases the barrier for other IDEs/Plugins to integrate Dafny support using the official language server easily.
Therefore, I am now at the point where I'll most-likely raise events in the language server that consumers such as the VSCode extension can handle to display active verifications.

tl;dr I try to integrate activity info as soon as possible.

**Comment by franck44** [comment]
@camrein 

> tl;dr I try to integrate activity info as soon as possible.

No urgency and thanks a lot for your help!

**Comment by camrein** [comment]
I have added the indicator to the language server and the extension. 

![image](https://user-images.githubusercontent.com/18049310/111795792-cdf22500-88c7-11eb-91d0-a6e2acc808f4.png)

Since there are discussions about including the language-server into the main repo of Dafny, it may take some time until I can release the update.
